### DISCUSSION TITLE
Installation without docker

### DISCUSSION THREAD

**Discussion by Dont-ask-for-it** [mealie-recipes/mealie#520] [Ideas]
I don't really like docker. A command line installation would be a great addition!

**Comment by hay-kot** [comment]
There is some general guidance on setting up the servers in the documentation https://hay-kot.github.io/mealie/documentation/getting-started/install/#deployed-without-docker

**Comment by Dont-ask-for-it** [comment]
Any development on this? :)

**Reply by hay-kot** [reply]
Other than [the above linked section](https://hay-kot.github.io/mealie/documentation/getting-started/install/#deployed-without-docker) in the documentation, I'm not going to provide anything else on this. 

If someone in the community wants to provide a guide I'm happy to include it in the documentation.

**Comment by olluz** [comment]
well, for someone to be able to provide a guide one would have to first be able to do it.
Unfortunately, the above linked section doesn't even provide half the information on what needs to be done. 
I do understand that you don't want to support any other installations besides docker and I respect that of course.
But people who have a problem with docker can't even start coming up with a detailed guide or installation script.

A bit more information would be very helpful. 

Thanks

**Reply by hay-kot** [reply]
Best I can offer you is to take a look at the docker file itself. It should be fairly self explanatory on what needs to happen to setup python and node.js. 

https://github.com/hay-kot/mealie/blob/mealie-next/docker/api.Dockerfile

**Reply by juancarbone** [reply]
What information are you talking about? It would be better if you wrote that there will be no support outside of DOCKER.

**Reply by juancarbone** [reply]
![Uploading image.png…]()

**Reply by Kuchenpirat** [reply]
That is the case. We do not recommend to install mealie outside of a container and we do not offer support for these types of setups.

It is certainly possible to install mealie outside of docker, but you would be on yourself to keep your instance and the node as well as python dependencies up to date.

**Comment by juancarbone** [comment]
I understand, but there are people who use other technologies and are
willing to keep the 3 instances you mentioned updated.
Thanks for the quick reply.

On Mon, Jul 22, 2024 at 5:02 PM Kuchenpirat ***@***.***>
wrote:

> That is the case. We do not recommend to install mealie outside of a
> container and we do not offer support for these types of setups.
>
> It is certainly possible to install mealie outside of docker, but you
> would be on yourself to keep your instance and the node as well as python
> dependencies up to date.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mealie-recipes/mealie/discussions/520#discussioncomment-10118946>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AESWXINM72KQDQP3WGHE5M3ZNVQPDAVCNFSM6AAAAABLI3EIE2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJRHA4TINQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 


*Juan Manuel Carbone*

**Reply by hay-kot** [reply]
> I understand, but there are people who use other technologies and are
willing to keep the 3 instances you mentioned updated.

if they’re capable of doing that then they should be capable of reading the docker file and translating that into deployment instructions.

**Comment by juancarbone** [comment]
I read that somewhere else. Thanks again.

On Mon, Jul 22, 2024 at 5:14 PM Hayden ***@***.***> wrote:

> I understand, but there are people who use other technologies and are
> willing to keep the 3 instances you mentioned updated.
>
> if they’re capable of doing that then they should be capable of reading
> the docker file and translating that into deployment instructions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mealie-recipes/mealie/discussions/520#discussioncomment-10119025>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AESWXIJ74MNMIBTFQCSTWOTZNVR3NAVCNFSM6AAAAABLI3EIE2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJRHEYDENI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 


*Juan Manuel Carbone*
